RiotSharp/SpectatorEndpoint/CurrentGame/BannedChampion.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using Newtonsoft.Json;
 
-namespace RiotSharp.CurrentGameEndpoint
+namespace RiotSharp.SpectatorEndpoint.CurrentGame
```

Comment:
Maybe I wasn't clear but I think you can flatten things and have only a `RiotSharp.SpectatorEndpoint` namespace and `SpectatorEndpoint` folder and have every class in this folder.

No need to have additional `CurrentGame` / `FeatureGames` folders and namespaces.

Sorry.

Reply:
I was unsure so I splitted in two but let's completely regroup.